### DISCUSSION TITLE
FreeTDS on Amazon RDS

### DISCUSSION THREAD

**Discussion by OleksandrAndrieianov** [FreeTDS/freetds#515] [Q&A] [answered]
Hello, 

I have Amazon RDS instance with MSSQL Server 2012 and Amazon RDS instance with PostgreSQL server.
I was able to connect MSSQL server tables to PostgreSQL DB via tds_fdw, but when I'm trying to run SELECT from mssql tables - I'm getting ERROR:

`ERROR [1:1]:(SQLSTATE: HV00L, SQLCODE: 0): ERROR: DB-Library error: DB #: 2404, DB Msg: Buffer overflow converting characters from client into server's character set, OS #: 0, OS Msg: Success, Level: 4`

From what I found in the internet - problem in collation, and I need to modify **freetds.conf** file, but the problem is that I cannot find how to do that in RDS. 
Can please someone help, how to modify **freetds.conf** file in Amazon RDS instance?

Thanks in advance!

**Comment by freddy77** [comment]
Not sure which kind of access you have to RDS. If you can ssh or similar should be just question of finding the file and open an editor on it. How did you install and configure `tds_fdw` ?

**Reply by OleksandrAndrieianov** [reply]
We have default admin access to RDS instance. 
I used tds_fdw on PostgreSQL server:
```

CREATE EXTENSION tds_fdw;

CREATE SERVER external_mssql
FOREIGN DATA WRAPPER tds_fdw 
OPTIONS (servername 'mssql_hostname, port '1433', database 'mssql_db_name', tds_version '7.4');
```

**Comment by freddy77** [comment, marked as answer]
Looking at https://github.com/tds-fdw/tds_fdw/blob/master/ForeignServerCreation.md there should be also a `character_set` option which you could try. I would try `UTF-8` value.

**Reply by OleksandrAndrieianov** [reply]
Yes, it worked! Thank you very much!
